backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ComponentDatabaseHandler.java

```diff
@@ -3025,6 +3025,23 @@ public List<ReleaseNode> getReleaseRelationNetworkOfRelease(Release release, Use
         getReleaseNodes(dependencyNetwork, user);
         return Collections.singletonList(dependencyNetwork);
     }
+    
+    public List<Release> releasesWithoutSRC(String id, User user) throws TException{
+    	ProjectService.Iface projectClient = new ThriftClients().makeProjectClient();
+    	Project project = projectClient.getProjectById(id, user);
+    	List<Release> releasesWithoutSRC = new ArrayList<>();
+    	Map<String, ProjectReleaseRelationship> releaseIdToUsage = project.getReleaseIdToUsage();
+    	Set<String> releaseIdsSet = releaseIdToUsage.keySet();
+    	List<String> releaseIdsList = new ArrayList<>(releaseIdsSet);
```

Comment:
Directly store the Map keys in list.
List<String> releaseIdsList = new ArrayList<>(releaseIdToUsage.keySet());